openbsd/OpenBSDProcessList.c

```diff
@@ -89,7 +90,7 @@ static inline void OpenBSDProcessList_scanMemoryInfo(ProcessList* pl) {
    size_t size = sizeof(uvmexp);
 
    if (sysctl(uvmexp_mib, 2, &uvmexp, &size, NULL, 0) < 0) {
-      CRT_fatalError("uvmexp sysctl call failed");
+      errx(1, "uvmexp sysctl call failed");
```

Comment:
`sysctl(3)` on OpenBSD seems to have a lot of possible failure conditions; why not use `err(3)` instead?